language-design.md

```diff
@@ -16,19 +14,29 @@ are:
 For each quote, describe what it was about the quote that led you pick it.
 
 **Response**
-
-
++ "The problem is that programming languages are created either by committee or by extreme technical wizards with magical math powers. The broad computer science academic community has not paid a tremendous amount of attention to programming language usability." (Pavlus, 2012)
+  + I absolutely agree with this statement made by Andreas Stefik, that is quoted in Pavlus' article. Since the first day I have started learning to program, I have felt like most of the programming languages are not user-friendly, or just targeted to be understood by only "technical wizards with magical math powers".
+  +  I certainly had this feeling when learning Haskell. It seemed as though everything about the language was created with a paradigm of category theory which must be understood (or worse, used in ignorance) to understand how to print (since I/O must occur through a Monad).  This is even more true when considering the Haskellian typeclasses, generics, and the type system. A solid understanding of functors, monoids, monads, and applicatives is expected by the designers. While Haskell is an extreme example, many programming languages expect substantial mathematical reasoning or domain knowledge.
```

Comment:
Our group chose the same quote as one of our three, and I very much agree with your sentiments. For example, in Haskell, why is Functor not called Mappable? The latter makes a lot more sense to someone from a non-mathematical background, while the former is *technically* correct, it's not as easy to think about.